packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -40,6 +40,7 @@ class TabBarTheme extends InheritedTheme with Diagnosticable {
     TabAlignment? tabAlignment,
     TextScaler? textScaler,
     TabIndicatorAnimation? indicatorAnimation,
+    BorderRadius? splashBorderRadius,
```

Comment:
IMO adding to this constructor isn't necessary, given that our goal is to transition these properties to `TabBarThemeData`.